static-assets/ng-views/audit.html

```diff
@@ -26,14 +26,17 @@ <h1>{{ 'admin.audit.AUDIT' | translate }}</h1>
             <span class="pr20" ng-show="!audit.sites">{{ audit.site }}</span>
             <div class="btn-group" uib-dropdown is-open="sitesDropdown.isopen" style="margin-right: 5px;" ng-show="audit.sites">
                 <button type="button" class="btn btn-default dropdown-toggle text-capitalize" uib-dropdown-toggle title="{{audit.site || 'admin.audit.ALL_SITES' | translate}}" >
-                    {{audit.site || 'admin.audit.ALL_SITES' | translate }}  <span class="caret"></span>
+                    {{audit.siteLabel || audit.site || 'admin.audit.ALL_SITES' | translate }}  <span class="caret"></span>
```

Comment:
Why are there so many different possibilities of what label to display? Is this an API inconsistency? Lets log a backend ticket if that’s the case.

Reply:
At this point we have 3 options.
1. All sites: This happen when we send site param empty to the api. We need to translate it.
2. Root Studio: Sumer wants to show to the user "System" label, instead of "Root Studio". So We need an extra label. Also we need to translate it
3. Sites (Other options): Each specific site selected by the user. 

So I'm trying to manage all those scenarios for it.

Reply:
Why is the value under two different properties, though? site or siteLabel

Reply:
Site is siteID from the API. It is displayed trough an angular repeat. Site label is a static value, in this case "System".